src/app/actions/city-list-actions/actions.tsx

```diff
@@ -0,0 +1,25 @@
+import { CityWeatherData } from "../../contracts/city-weather-data";
+
+export class SubmitFavorite {
+    constructor(private data: CityWeatherData) {
+    }
+    public get Data(): CityWeatherData {
+        return this.data;
+    }
+}
+export class DeleteFavorite {
+    constructor(private city: string) {
```

Comment:
Suggest use more targeted name for variables. `city` -> `cityName` (depends on situation).